src/System.Buffers.Experimental/System/Buffers/ReadOnlyBytes.cs

```diff
@@ -177,6 +179,24 @@ int IndexOfRest(byte value, int firstLength)
             return -1;
         }
 
+        public Cursor CursorOf(byte value)
+        {
+            ReadOnlySpan<byte> first = _first.Span;
+            int index = first.IndexOf(value);
+            if (index != -1) return new Cursor(this, index);
```

Comment:
Allocation in most common case?

Reply:
Where? `Cursor` is a struct.

Reply:
Ah, now I see. Thanks.

Reply:
I will fix it. Null list will mean first.